frontend/src/page/Player.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+
+import { graphql, useLazyLoadQuery } from "react-relay/hooks";
+import { PlayerQuery } from "../query-types/PlayerQuery.graphql";
+
+
+type Props = {
+    id: string;
+};
+
+export const PlayerPage: React.FC<Props> = ({ id }) => {
+    const { event } = useLazyLoadQuery<PlayerQuery>(
+        graphql`
+            query PlayerQuery($id: ID!) {
+                event(id: $id) {
+                    title
+                    video
+                    description
+                }
+            }
+        `,
+        { id: `ev${id}` },
```

Comment:
Mh and we have to think about this in the long term I guess? Dunno, seems maybe not ideal to have these prefix duplicated across frontend code? Dunno. Maybe it's not that bad. Mh.

Reply:
Yeah, I thought that as well, but what's the solution? Compile our `id` Rust module to WASM and import/use it here? :thinking:

Reply:
Yeah no that doesn't sound great either. Maybe this is actually the only place where one needs to add `ev`. So yeah, let's ignore that until it actually becomes problematic :P